flake.nix

```diff
@@ -155,6 +155,8 @@
           src = filterWorkspaceDepsBuildFiles ./.;
           pname = "fedimint-dependencies";
           doCheck = false;
+          # we do not care about running `cargo check`
+          cargoCheckCommand = "true";
```

Comment:
Slightly unintuitive why `true` deactivates it. For others who are trying to figure it out: this replaces the `cargo check` command normally used (I guess) with the `true` command, which just always terminates with exit code 0 (success).